src/shared/parameter_v2/config_definitions/passing_config.yaml

```diff
@@ -0,0 +1,119 @@
+ - double:
+     name: static_field_position_quality_x_offset
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+                The offset from the sides of the field to place the rectangular
+                sigmoid we use to determine what areas to pass to
+ - double:
+     name: static_field_position_quality_y_offset
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+                The offset from the sides of the field to place the rectangular
+                sigmoid we use to determine what areas to pass to
+ - double:
+     name: static_field_position_quality_friendly_goal_distance_weight
+     min: 0
+     max: 1
+     value: 0.3
+     description: >-
+         The weight that being close to the goal will have on the static
+         position quality. Lower, more negative weights result in the distance
+         to the goal having less of an effect.
+ - double:
+     name: enemy_proximity_importance
+     min: 0
+     max: 10
+     value: 0.5
+     description: >-
+                This controls how heavily we weight an enemy robot being near the pass
```

Comment:
Nit: I think weight is a noun and weigh is a verb.

Reply:
fixed